src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -899,7 +899,7 @@ def check_error_and_log(error_regex, log_message_format):
 
           elif (num_first_testcase_hangs ==
                 self.MAX_FIRST_HANGS_WITH_DEFERRED_FORKSERVER):
-            environment.set_value(constants.DONT_DEFER_FORKSERVER, 1)
+            environment.set_value(constants.AFL_DRIVER_DONT_DEFER, 1)
```

Comment:
I think this is still DONT_DEFER_ENV_VAR ?

Reply:
Oh wow, I messed this PR up. Good catch.

Reply:
Fixed.